### DISCUSSION TITLE
Siri support?

### DISCUSSION THREAD

**Discussion by zstonez** [hassio-addons/app-airsonos#119] [General]
Hi,
i just found that addon and i got it working. Now i can see my old play 1 or 5 in AirPlay when i olay music with my iphone for instanze. Im just wondering because when i try siri to play music on this device, it doesnt find the speaker. Is there something wrong in my understanding or i there just no siri support?

**Comment by frenck** [comment]
That is more a question for apple. There is nothing in there that is specifically for Siri (or not). It exposes speakers. Siri being able/not able to control those speakers isn't related or relevant for this add-on at all 🤷
